src/didactUriCompletionItemProviderAsciiDoc.ts

```diff
@@ -21,52 +21,53 @@ import * as vscode from 'vscode';
 import {didactProtocol, DidactUriCompletionItemProvider} from './didactUriCompletionItemProvider';
 import * as extensionFunctions from './extensionFunctions';
 
+// TODO: Figure out how to redo these options
```

Comment:
Why it needs to be "redone"?

it sounds to me that we can keep both approaches at the same time.

Reply:
Just wanted to give myself the opportunity to retool these options a bit since we're doing so much with the other half of the equation. We may very well leave these options intact.